google/cloud/firestore_v1/query.py

```diff
@@ -333,16 +339,17 @@ def order_by(self, field_path, direction=ASCENDING):
             field_filters=self._field_filters,
             orders=new_orders,
             limit=self._limit,
+            limit_to_last=self._limit_to_last,
             offset=self._offset,
             start_at=self._start_at,
             end_at=self._end_at,
             all_descendants=self._all_descendants,
         )
 
     def limit(self, count):
-        """Limit a query to return a fixed number of results.
+        """Limit a query to return the next `count` matching results.
```

Comment:
Slight rephrasing to be more inline with how the query functions:

```suggestion
        """Limit a query to return at most `count` matching results.
```